bom/application/pom.xml

```diff
@@ -201,6 +201,7 @@
         <webjars-locator-core.version>0.46</webjars-locator-core.version>
         <log4j2-jboss-logmanager.version>1.0.0.Beta1</log4j2-jboss-logmanager.version>
         <avro.version>1.10.0</avro.version>
+        <gradle-tooling.version>6.5</gradle-tooling.version>
```

Comment:
Looks like this property could be removed from this pom. But let's see if there are other things to change in this PR first.